peerbanhelper-privacy-zh-CN.md

```
@@ -29,9 +29,22 @@ PBH-BTN 团队是由来自五湖四海各行各业的热爱 BitTorrent 技术的
 
 ## 我们收集的信息
 
-PeerBanHelper 作为一个注重隐私的开源项目，绝不会滥用自己的权限，收集用户额外信息，目前 PeerBanHelper 获取并传输的所有数据如下：
+PeerBanHelper 作为一个注重隐私的开源项目，绝不会滥用自己的权限，收集用户额外信息。秉承透明、公开、可控、正当使用原则向您披露我们对收集的信息如何处理、使用。  
+目前 PeerBanHelper 获取并传输的所有数据如下：
 
-> 当前版本默认安装下的 PeerBanHelper 不向任何人发送任何数据。
+* 您的匿名安装标识符、PeerBanHelper 的版本号，以及操作系统类型与版本号等基本信息
+  * 我们用此数据来确定目前有多少用户在使用我们的程序，及其版本分布情况。同时，我们也用此数据来了解用户所使用的操作系统类型、指令集、与版本分布情况，以便我们在开发过程中进行针对性优化
+* 程序发生故障时，收集的错误信息和错误上下文
+  * 我们收集、使用这些数据用于分析程序故障原因，并对其实施修复以提升程序稳定性
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)